conjure-java-jersey-server/src/main/java/com/palantir/conjure/java/server/jersey/RemoteExceptionMapper.java

```diff
@@ -54,64 +54,35 @@
 
     @Override
     public Response toResponseInner(RemoteException exception) {
-        ErrorType errorType = mapErrorType(exception);
-        Response.ResponseBuilder builder = Response.status(errorType.httpErrorCode());
-
-        try {
-            SerializableError error = SerializableError.builder()
-                    .errorName(errorType.name())
-                    .errorCode(errorType.code().toString())
-                    .errorInstanceId(exception.getError().errorInstanceId())
-                    .build();
-            builder.type(MediaType.APPLICATION_JSON);
-            builder.entity(error);
-        } catch (RuntimeException e) {
```

Comment:
These catch blocks appear to be a holdover from when we performed the serialization manually in the exception mapper. This was changed in https://github.com/palantir/conjure-java-runtime/pull/675 and these error messages are now misleading/incorrect.

This code should never throw exceptions, so these catch blocks can be removed.